### PR TITLE
I could blot the sun out with these titty pills and some certain catgirls here STILL wouldn't be satisfied.

### PR DESCRIPTION
**Pull request by BlueWildrose**
## About The Pull Request

This does two things:
1. Doubles the amount of breast and penis enlargement pill bottles in the Kinkmate vendor.
2. Increases the pills in each pill bottle from 7 to 10.

This effectively means that there's 1,000 units of succubus milk and incubus draft inside a kinkmate.

And before someone asks, that means it's now reduced in value for cargo. Yes, I thought ahead there.

## Why It's Good For The Game

These pills seem to be in high demand for some reason. I wonder why. The people behind Kinkmate probably would increase their supply to fit the demand the best they can.

Also I know I made this PR on April Fools but I'm actually serious about this PR.

## Changelog
:cl:
add: There are now three more pills in each breast or penis enlargement pill bottle.
tweak: There are now 10 pill bottles of breast and penis enlargement pill bottles in a Kinkmate instead of 5.
balance: Because of such changes that increases their amount in the kinkmate, succubus milk and incubus draft values are reduced to RARE from VERY_RARE.
spellcheck: Grammar correction on titty pill bottle.
/:cl:

### CODE REVIEW

**Comment by The-Sun-In-Splendour** [comment]
> And before someone asks, that means it's now reduced in value for cargo. Yes, I thought ahead there.

Another fucking cargo nerf for your cringe fetish needs, close this pr right the fuck now there's like 500 kinkmates on station

**Comment by Detective-Google** [comment]
me and the boys in chem rushing the kindvend to buy all the titty skittles and dump them out for the powergame bottles 😎

**Review by LetterN — APPROVED** [review]
yes

**Comment by BlueWildrose** [comment]
> 
> 
> > And before someone asks, that means it's now reduced in value for cargo. Yes, I thought ahead there.
> 
> Another fucking cargo nerf for your cringe fetish needs, close this pr right the fuck now there's like 500 kinkmates on station

>order kinkmate restocking units from cargo
>buy pills from kinkmate
>throw pills into cargo ship
>???
>profit

granted there's the factor that you're spending your credits in return for funding the station but still this is something I'd rather not risk

**Comment by The-Sun-In-Splendour** [comment]
> 
> 
> > > And before someone asks, that means it's now reduced in value for cargo. Yes, I thought ahead there.
> > 
> > 
> > Another fucking cargo nerf for your cringe fetish needs, close this pr right the fuck now there's like 500 kinkmates on station
> 
> > order kinkmate restocking units from cargo
> > buy pills from kinkmate
> > throw pills into cargo ship
> > ???
> > profit
> 
> granted there's the factor that you're spending your credits in return for funding the station but still this is something I'd rather not risk

basic economics
get fucked

**Comment by BlueWildrose** [comment]
yeah you know what, just because of the funny factor that you're actually spending your own credits in order to make a profit for the station, I'll increase it to RARE.

Under its old value of VERY_RARE though, you could've made an 7,200 credit profit off of buying a single kinkmate restocking unit.

(You have to spend 800 credits total from your OWN account to buy out all the pills, and the kinkmate restocking costs 2,000 credits.)

With it being changed to RARE, you would make a profit of 2,200 credits.

It's still probably not worth draining your account over. Go cut some wood and glass instead.

**Comment by LordSaladin** [comment]
This is a juri buff, now all we need is hyper's sprites and juri will have an entire fanservicey anime plot's worth of oppai

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -394,15 +394,15 @@
 	desc = "You want penis enlargement pills?"
 
 /obj/item/storage/pill_bottle/penis_enlargement/PopulateContents()
-	for(var/i in 1 to 7)
+	for(var/i in 1 to 10)
 		new /obj/item/reagent_containers/pill/penis_enlargement(src)
 
 /obj/item/storage/pill_bottle/breast_enlargement
 	name = "breast enlargement pills"
 	desc = "Made by Fermichem - They have a woman with breasts larger than she is on them. The warming states not to take more than 10u at a time."
```
Comment (HeroWithYay) [inline code/game/objects/items/storage/firstaid.dm]: Can you correct line 402 to say:
**"They picture a woman sporting breasts much larger than her own body. A warning label advises not to take more than 10u at a time."**
Spelling mistake + poor sentence structure.
It irks me.

Comment (BlueWildrose) [reply code/game/objects/items/storage/firstaid.dm]: Done

**Pull request merged**